instructionAPI/src/decoder/riscv/opcode_xlat.C

```diff
@@ -0,0 +1,668 @@
+#include "capstone/riscv.h"
+#include "entryIDs.h"
+#include "opcode_xlat.h"
+
+namespace Dyninst {
+namespace InstructionAPI {
+namespace riscv {
+
+// clang-format off
+
+  // Use the entryID that Capstone has and ignore aliases found in entryIDs.h
+  entryID translate_encoded_opcode(riscv_insn id) {
```

Comment:
Is there a reason to have both `translate_encoded_opcode` and `translate_opcode`?  If not, just combine them.  A more descriptive name would be `capstone_opcode_to_entryID`